### PR TITLE
add dependency links for private vsc-* dependencies

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
this started working as we like since setuptools 27.0, so update your setuptools if you now can't find dependencies in tests. (no need to any longer suply dependency links in setup.py for our vsc-* projects thare are hosted on github)

File: lib/vsc/install/shared_setup.py [hunk 1]
```diff
@@ -1328,7 +1328,30 @@ def parse_target(self, target, urltemplate=None):
...
+            ]
+        else:
+            urls = [('github.com', 'git+https://')]
+        for dependency in set(new_target['install_requires'] + new_target['setup_requires'] +
+            new_target['tests_require']):
+            if dependency.startswith('vsc'):
+                dep = dependency.split(' ')[0]
+                depversion = ''
+                for comp in ['=', '<']:
+					try:
```
Comment (stdweird) [inline lib/vsc/install/shared_setup.py]: moar whitespace

**Comment by stdweird** [comment]
@JensTimmerman version bump pls

**Comment by boegel** [comment]
@JensTimmerman we don't have a style check for (well, against) tabs?

**Comment by JensTimmerman** [comment]
@boegel prospector tests all sorts of things, we currently don't have everything turned on yet because it would probably be to annoying to have to rework all the code, however, maybe we should, because every check has a good reason, and as we turn on more and more over time it becomes annoying to update all code a little bit every time...
anyway, turned on check for mixing tabs and spaces.

**Pull request merged**